### DISCUSSION TITLE
How to set the step boxes in the flowchart to be the same width?

### DISCUSSION THREAD

**Discussion by songyuc** [mermaid-js/mermaid#4043] [Q&A]
Hi, guys,
I am learning mermaid, and currently drawing a flowchart with it.
Here is the code:
```js
flowchart TD;
Aa(Aa) --> Bbbb(Bbbb);
Bbbb(Bbbb) --> Ccccccc(Ccccccc);
```
Output:
![image](https://user-images.githubusercontent.com/27288110/215334445-be15a4ae-ef1e-4c40-9a25-5e1cb6f1fadf.png)
And I want the those step boxes to a same width to the longest width of all the step boxes.
So how can I realize it?

Your answer and guide will be appreciated!

**Comment by elzup** [comment]
seems to be unsupported.

node width can style as below. but broken layout on simple flowchart.

```mermaid

flowchart TD;
Aa(Aa) --> Bbbb(Bbbb);
Bbbb(Bbbb) --> Ccccccc(Ccccccc) --> Ddd;

classDef node width:100px;
```

other refs
flowchart: add minNodeWidth, minNodeHeight · Issue #3773 

I think the dynamic max size solution is further down the road.
or self padding by max char length.
